tiup/tiup-dm-topology-reference.md

```diff
@@ -0,0 +1,293 @@
+---
+title: Topology Configuration File for DM Cluster Deployment using TiUP
+---
+
+# Topology Configuration File for DM Cluster Deployment using TiUP
+
+To deploy or scale a TiDB Data Migration (DM) cluster, you need to provide a topology file to describe the cluster topology. Similarly, to modify the cluster topology, you need to modify the topology file. The difference is that you can only modify a part of the fields in the topology file.
+
+Here is an [example topology file](https://github.com/pingcap/tiup/blob/master/embed/templates/examples/dm/topology.example.yaml) for your reference.
+
+## File Structure
+
+The topology file of a DM cluster might contain the following blocks:
```

Comment:
Refer to https://github.com/pingcap/docs/pull/5314/files#diff-33150822cf42e37369ebf2cd8dec35142aefda117ef5015af5755e6465d56d02R13
```suggestion
The topology file of a DM cluster might contain the following sections:
```